xbstrap/base.py

```diff
@@ -618,7 +622,10 @@ def get_remote_commit(ref):
 		elif 'hg' in self._this_yml:
 			if not os.path.isdir(self.source_dir):
 				return ItemState(missing=True)
-			args = ['hg', 'manifest', '--pager', 'never', '-r',]
+#DCC			args = ['hg', 'manifest', '--pager', 'never', '-r',]
+# my hg complained heavily of --pager not being recognized - don't know if this is local problem...
```

Comment:
This is unrelated to the remainder of the PR (but should be investigated).

Reply:
This breaks my on local system... any ideas which hg I should download and use? I just used ubuntu packages

Reply:
removing - turns out PATH was using older 3.6.2 hg from a different project - sorry